ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c

```diff
@@ -465,10 +533,16 @@ _set_max_detection (bounding_boxes * data, const guint max_detection,
  * Both tensors are MANDATORY!
  *
  * [Tensorflow SSD Model]
- * The first tensor is num_detection. 1, ANY-TYPE
- * The second tensor is detection_classes. #MaxDetection, ANY-TYPE
- * The third tensor is detection_scores. #MaxDetection, ANY-TYPE
- * The fourth tensor is detection_boxes. BOX_SIZE : #MaxDetection, ANY-TYPE
+ * Tensors mapping is defined through option-3, with following syntax:
+ * LOCATIONS_IDX:CLASSES_IDX:SCORES_IDX:NUM_DETECTION_IDX
+ *
+ * Default configuration is: 3:2:1:0
```

Comment:
Doesn't this configuration incur regression errors for deployed applications & unit-tests (the reason why CI's automated test cases of x64 are failing?)
The default configuration should be the configuration of previous versions. (would it be 3:1:2:0 ?)

Reply:
Thanks for your review.

I don't think this tensor mapping related change introduced the regression ( I checked this behaved the same without it on my board as this was failing..)

I would have rather thought to the changes related to valid class (0) and the applied threshold on the detected objects, which lead obviously to different decoder outputs.

But I am a bit confused as the decoder bounding boxes unittests passed on my device (aarch64, yocto zeus)..

I guess I have to reproduce the issue on an ubuntu build (x86_64) to progress further as the TAOS output logs is .. minimalist.

In the meantime, I might update the PR with less functional changes to narrow down what is wrong.

Reply:
You were finally right, I made a stupid typo while coding the default values.. my attention was on first and last items of the tensors indexes :(

The threshold was causing issues as well to pass unit test, so the threshold can be set through option-3, and if not specified, all detected objects will be output so backward compatibility is preserved.